vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/HttpRequestSteps.java

```diff
@@ -68,9 +71,12 @@ public HttpRequestSteps(HttpTestContext httpTestContext, HttpRequestExecutor htt
      * @param content HTTP method request body
      */
     @Given("request body: $content")
-    public void request(String content)
+    public void request(DataWrapper content)
```

Comment:
please update Javadoc and docs to mention that user is responsible for content-type header provision